include/boost/json/string.hpp

```diff
@@ -1871,6 +2477,31 @@ class string
         return append(s.data(), s.size());
     }
 
+    /** Append characters from an object convertible to `string_view`.
+
+        Constructs a temporary `string_view` object `sv` from `t`
+        and appends `sv.substr(pos_str, count)` to the end of the string.
+
+        @par Exception Safety
+
+        Strong guarantee.
+
+        @par Constraints
+
+        `T` satisfies __is_string_viewish__.
```

Comment:
what is `__is_string_viewish__` did I write that? This is a qbk macro (check main.qbk to find the definition), please make sure it expands to something public.

Reply:
I'll expand it to what the standard specifies (convertible to string_view, not convertible to const char*)

Reply:
Is that what my code does?